src/actions/profileAction.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable no-unused-vars */
+import { types } from 'util';
```

Comment:
I don't think you need this line since you've already imported the action types. Otherwise, this is nice work @unah254

Reply:
Thanks, I've implemented the feedback